mqt/benchviewer/src/backend.py

```diff
@@ -7,10 +7,14 @@
 
 import pandas as pd
 import requests
-from importlib import metadata
 from tqdm import tqdm
 from packaging import version
 
+try:
+    from importlib import metadata
+except ImportError:
+    import importlib_metadata as metadata
```

Comment:
I'd rather wrap this in a similar 
```
if sys.version_info < (3, 10, 0):
...
```
block as the `importlib.resources` import since that is how the dependency in the `setup.py` file is defined and since `importlib.metadata` is no longer provisional since Python 3.10.

Reply:
done